capstone/src/main/webapp/details.js

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Sets the page details based on the restaurant
+ * stored in the session.
+ */
+function setRestaurantDetails() {
+  // Retrieve stored restaurant from session
+
+  const currRestaurant =
+      JSON.parse(sessionStorage.getItem('selectedRestaurant'));
+
+  // Set UI elements
+  if (currRestaurant !== null && currRestaurant !== '') {
+    if (document.getElementById('title') !== null) {
+      document.getElementById('title').innerText = currRestaurant.name;
+    }
+    if (document.getElementById('story') !== null) {
+      document.getElementById('story').innerText = currRestaurant.story;
+    }
+    if (document.getElementById('website') !== null) {
+      document.getElementById('website').innerHTML = '<a href=\'' +
+          currRestaurant.website + '\'>' + currRestaurant.website + '</a>';
```

Comment:
Try to avoid putting raw HTML. As a solution, we can simply add the <a> by creating it with a method and then appending it to the 'website' element as a child.

Reply:
Thanks for the tip, should be updated correctly!